src/common/Database.js

```diff
@@ -183,7 +183,13 @@
     };
 
     Grid.prototype.fieldByLabel = function (_, ignoreCase) {
-        return this.fields().filter(function (field, idx) { field.idx = idx; return ignoreCase ? field.label().toLowerCase() === _.toLowerCase() : field.label() === _; })[0];
+        var retVal = this.fields().filter(function (field, idx) { field.idx = idx; return ignoreCase ? field.label().toLowerCase() === _.toLowerCase() : field.label() === _; })[0];
+        if (!retVal) {
+            console.log("Not able to find field: " + _);
+            console.log(this.fields());
+            return {};
```

Comment:
then we need to change every place that function is used so that when its looking for {}.idx it throws a console log and does a check ... just seems a bit redundant when we can handle it in one place